grammars.yml

```diff
@@ -1176,9 +1180,6 @@ vendor/grammars/vscode-singularity:
 vendor/grammars/vscode-slice:
 - source.ice
 - source.slice
-vendor/grammars/vscode-vba:
-- source.vba
-- source.wwb
```

Comment:
You didn't want to do this 😁

Reply:
I just ran the script, no idea how that happened, reverted 👍